scripts/doc_templates/templates/protex

```diff
@@ -297,7 +296,7 @@ LINE:
 #       Straight quote
 #       --------------
         if ($Fld[0] eq '!QUOTE:') {
-           for ($i = 1; $i <= $#Fld - 1; $i++) {
+           for ($i = 1; $i <= $#Fld; $i++) {
```

Comment:
I read the Perl documentation. `$#array` is the last index of an array. Good catch.